scripts/test262-runner.js

```diff
@@ -1205,6 +1205,14 @@ function filterDescription(data: BannerData): boolean {
     !data.description.includes("iterable");
 }
 
+function filterCircleCI(data: BannerData): boolean {
+  let skipTests = ['7.8.5_A1.4_T2', '7.8.5_A2.4_T2', '7.8.5_A2.1_T2', '7.8.5_A1.1_T2',
+                    '15.1.2.2_A8', '15.1.2.3_A6', '7.4_A6'];
+
+  if (process.env.CIRCLECI && skipTests.indexOf(data.es5id) > -1) return false;
```

Comment:
Instead of checking for the CIRCLECI variable, just exclude these tests on all platforms, but make it conditional on another variable so that nightly runs can include them.